lib/cogctl/actions/users/util.ex

```diff
@@ -8,9 +8,16 @@ defmodule Cogctl.Actions.Users.Util do
 
   def render(user_info, group_info, role_info, sort) do
     Table.format(user_info, false)
-        <> "\n\nGroups:\n" <> Table.format(group_info, sort)
-        <> "\n\nRoles:\n" <> Table.format(role_info, sort)
+        <> "\n\nGroups:\n" <> render_groups(group_info, sort)
+        <> "\n\nRoles:\n" <> render_roles(role_info, sort)
```

Comment:
It's always a little tough to parse literal `\n` in here. Could be cool to use here docs like:

```
display_output("""
  #{Table.format(user_info, false)}

  Groups:
  #{render_groups(group_info, sort)}

  Roles:
  #{render_roles(role_info, sort)}
""")
```